dcos_launch/cli.py

```diff
@@ -33,9 +33,10 @@
 """
 import os
 import sys
+
 import dcos_launch
-from dcos_launch import util
 import dcos_launch.config
+from dcos_launch import util
```

Comment:
Yay PEP8! ;)

Reply:
@falseperfection Actually this is the result of something you told me to do on my first PR at Mesosphere: isort